src/cleanvision/imagelab.py

```diff
@@ -289,15 +294,26 @@ def _visualize(
 
     def visualize(
         self,
-        issue_types: List[str],
+        filepaths: Optional[List[str]] = None,
+        issue_types: Optional[List[str]] = None,
+        num_images: int = 4,
         examples_per_issue: int = 4,
         cell_size: Tuple[int, int] = (2, 2),
     ) -> None:
-        for issue_type in issue_types:
-            self._visualize(issue_type, examples_per_issue, cell_size)
+        if issue_types:
```

Comment:
TODO: documentation should clarify `issue_types` supersedes `filepaths` when both are specified